canvas_modules/harness/test_resources/parameterDefs/structurelisteditor_paramDef.json

```diff
@@ -1216,7 +1216,7 @@
       },
       {
         "complex_type_ref": "inlineEditingTableError",
-        "row_selection": "single",
+        "row_selection": "multiple-edit",
```

Comment:
This change is not needed. You can revert this to "single". Table looks odd because there is no header for column which has delete icons.
![Screenshot 2024-10-30 at 5 32 22 PM](https://github.com/user-attachments/assets/a54452dd-6c4d-42a6-bc7e-221e0228a771)


Take a look at how the header is added for subpanel column here - https://github.com/elyra-ai/canvas/blob/main/canvas_modules/common-canvas/src/common-properties/controls/abstract-table.jsx#L705
![Screenshot 2024-10-30 at 5 34 31 PM](https://github.com/user-attachments/assets/5b5df857-7690-46be-85d6-ff9e3372a268)


You should add a similar header for single select tables which will have a delete rows column.

Reply:
I switched from single to multi select because single tables no longer have the toolbar which means that single tables can no longer move rows. I discussed this with Matt and Jesus, and they mentioned to switch it to multi select to ensure test coverage.